File: code_examples/4-Functions/3-Return_Values/Python/main.py
#!/usr/bin/env python3
"""
Created by: Mr. Coxall
Created on: Sep 2020
This module calculates the area of circle
"""

import math

def calculate_area(radius: int) -> float:
    """The calculate_area() function calculates the area of circle, returns None."""

    # process
    area = math.pi * radius ** 2

    return area


def main() -> None:
    """The main() function just calls other functions, returns None."""

    # input
    radius = int(input("Enter the radius of a circle (cm): "))
    print("")

    # call functions
    area = calculate_area(radius)

    # output
    print(f"The area is {area:.2f} cm²")

    print("\nDone.")


if __name__ == "__main__":
    main()

File: code_examples/4-Functions/3-Return_Values/CSharp/main.cs
/* Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program calculates the area of circle
*/

using System;

/*
 * The Program class
 * Contains all methods for performing basic variable usage
*/
class Program {
    static float calculateArea(int radius) {
        // this function calculates the area of circle

        // process
        float area = Math.Pi * radius ** 2;


    }
 
    public static void Main (string[] args) {
        int radius;
        float area = 0.0;

        // input
        Console.Write("Enter the radius of a circle (cm): ");
        radius = int.Parse(Console.ReadLine());
        Console.WriteLine("");

        // call functions
        area = calculateArea(length, width);

        // output
        Console.WriteLine($"The area is {area} cm²");

        Console.WriteLine("\nDone.");
    }
}
